project/Dependencies.scala

```diff
@@ -41,22 +43,24 @@ object Dependencies {
   )
 
   lazy val example = Seq(
-    "org.typelevel"                %% "cats-core"                     % "2.6.1",
-    "io.circe"                     %% "circe-generic"                 % "0.14.1",
-    "org.http4s"                   %% "http4s-core"                   % Versions.http4s,
-    "org.http4s"                   %% "http4s-blaze-server"           % Versions.http4s,
-    "org.http4s"                   %% "http4s-dsl"                    % Versions.http4s,
-    "org.http4s"                   %% "http4s-circe"                  % Versions.http4s,
-    "io.jaegertracing"              % "jaeger-core"                   % Versions.jaeger,
-    "io.jaegertracing"              % "jaeger-client"                 % Versions.jaeger,
-    "io.jaegertracing"              % "jaeger-zipkin"                 % Versions.jaeger,
-    "com.github.pureconfig"        %% "pureconfig"                    % "0.16.0",
-    "com.softwaremill.sttp.client" %% "async-http-client-backend-zio" % Versions.sttp,
-    "com.softwaremill.sttp.client" %% "circe"                         % Versions.sttp,
-    "dev.zio"                      %% "zio-config"                    % Versions.zioConfig,
-    "dev.zio"                      %% "zio-config-magnolia"           % Versions.zioConfig,
-    "dev.zio"                      %% "zio-config-typesafe"           % Versions.zioConfig,
-    "dev.zio"                      %% "zio-interop-cats"              % Versions.zioInteropCats
+    "org.typelevel"                 %% "cats-core"                     % "2.6.1",
+    "io.circe"                      %% "circe-generic"                 % "0.14.1",
+    "org.http4s"                    %% "http4s-core"                   % Versions.http4s,
+    "org.http4s"                    %% "http4s-blaze-server"           % Versions.http4s,
+    "org.http4s"                    %% "http4s-dsl"                    % Versions.http4s,
+    "org.http4s"                    %% "http4s-circe"                  % Versions.http4s,
+    "io.jaegertracing"               % "jaeger-core"                   % Versions.jaeger,
+    "io.jaegertracing"               % "jaeger-client"                 % Versions.jaeger,
+    "io.jaegertracing"               % "jaeger-zipkin"                 % Versions.jaeger,
+    "com.github.pureconfig"         %% "pureconfig"                    % "0.16.0",
```

Comment:
Same here.

Reply:
`JaegerTracer` still depends on the jaegertracing library. Am I supposed to use something else instead?

Reply:
I'm referring to pureconfig, not to the tracer. If it's used in other example, you can remove it later.